docs/explanation/gno-tooling/cli/gnokey.md

```diff
@@ -179,7 +179,7 @@ gnokey maketx addpkg \
 
 ### `call`
 
-This subcommand lets you call a public function.
+This subcommand lets you call a public function on a Realm.
```

Comment:
Surprisingly, you can also call a public function of a package like `gno.land/p/xxx`.

Reply:
Good catch. You can call functions to just return some pure values, ie. 
```
func EchoNumber(num int) int {
      return num
}
```

I'll remove this piece of text.